client/webserver/site/src/js/wallets.ts

```diff
@@ -199,13 +201,39 @@ export default class WalletsPage extends BasePage {
     // amount field.
     bind(page.sendAvail, 'click', () => {
       const asset = this.sendAsset
-      page.sendAmt.value = String(asset.wallet.balance.available / asset.info.unitinfo.conventional.conversionFactor)
+      const bal = asset.wallet.balance.available
+      const unitInfo = asset.info.unitinfo
+      const rate = app().fiatRatesMap[asset.id]
+      if (rate && page.sendValue.firstElementChild) page.sendValue.firstElementChild.textContent = Doc.formatAssetValue(bal, rate, unitInfo)
+      else Doc.hide(page.sendValue)
+      page.sendAmt.value = String(bal / unitInfo.conventional.conversionFactor)
       // Ensure we don't check subtract checkbox for assets that don't have a
       // withdraw method.
       if ((asset.wallet.traits & traitWithdrawer) === 0) page.subtractCheckBox.checked = false
       else page.subtractCheckBox.checked = true
     })
 
+    for (const [assetID, wallet] of (Object.entries(app().walletMap) as [any, WalletState][])) {
+      if (!wallet) continue
+      const rate = app().fiatRatesMap[assetID]
+      const fiatDiv = this.page.walletTable.querySelector(`[data-conversion-target="${assetID}"]`) as PageElement
+      if (!fiatDiv || !fiatDiv.firstElementChild) continue
+      fiatDiv.firstElementChild.textContent = Doc.formatAssetValue(wallet.balance.available, rate, app().unitInfo(assetID))
+      if (rate && !app().disableConversion) Doc.show(fiatDiv)
+      else Doc.hide(fiatDiv)
```

Comment:
This block is repeated 3 times in `MarketsPage` methods. How about a little helper method.